packages/medusa-react/test/utils.tsx

```diff
@@ -48,17 +55,17 @@ export function createCartWrapper() {
     initialBagState,
     initialCartState,
   }: {
-    initialBagState?: BagState
+    initialBagState?: SessionCartState
     initialCartState?: Cart
     children?: React.ReactNode
   }) => {
     return (
       <MedusaProvider queryClientProviderProps={{ client: qc }} baseUrl="">
-        <BagProvider initialState={initialBagState}>
+        <SessionCartProvider initialState={initialBagState}>
```

Comment:
Three cases of `initialBagState` here. Should be changed to `initialSessionCartState`.